### PR TITLE
C2 mohamed activitylog

### PR DESCRIPTION
**Pull request by mohamedahmedawad93**
@mahmoudHussein @HebaElSobky please review

### CODE REVIEW

**Comment by mahmoudHussein** [comment]
@mohamedahmedawad93 please hook this request to the issue

**Comment by mohamedahmedawad93** [comment]
#210

File: project/tager/tager_www/views.py [hunk 1]
```diff
@@ -979,9 +1036,57 @@ def intrested(request):
...
+        #post_log_type is the type of the log type the user will choose in the activity log page
+        post_activity_content = "you added " + unicode(post_in.title) + " to your wish list."
+        post_activity_url = "showpost?post=" + unicode(post_in.id)
+        post_log_type = "profile"
+        # post_log_date = datetime.datetime.now()
+        log = ActivityLog.objects.create(content = post_activity_content, url = post_activity_url, log_type = post_log_type, user = user)
 
     return HttpResponse()
 
+def all_log(request):
```
Comment (mahmoudHussein) [inline project/tager/tager_www/views.py:1167]: @mohamedahmedawad93 please when you are merging with the new master add documentation for these methods.

**Comment by mohamedahmedawad93** [comment]
@mahmoudHussein i have added my documentations #210

**Comment by HebaElSobky** [comment]
@mohamedahmedawad93 i cant view anything fl activity log l fl profile when i edit my name ??

**Comment by abdelrahmanMaged** [comment]
@HebaElSobky it should work try to pull the branch again

**Comment by abdelrahmanMaged** [comment]
@HebaElSobky please review again urgently

**Comment by HebaElSobky** [comment]
@mohamedahmedawad93 this is the right branch right?

**Comment by HebaElSobky** [comment]
@AbdelrahmanAyad  @mohamedahmedawad93 scenario is verified

**Pull request merged**